### DISCUSSION TITLE
文献在文内引用，没有按照年份排列，按照拼音顺序排列了

### DISCUSSION THREAD

**Discussion by earthmoon** [whutug/whu-thesis#121] [Q&A]
你好，文献在文内引用，没有按照年份排列，按照拼音顺序排列了。比如同时引用两位作者的文献，在文字的引用处，是按照拼音排列的，并不是年份排列。采用的是Biblatex

**Comment by sikouhjw** [comment]
请你提供『需要按年份排列』的依据。

**Reply by earthmoon** [reply]
没有依据，英文一般也是按字母顺序排列。是我之前的习惯吧。
